pagopa/pn-frontend
# make large list of organizations in delegations to be visible in scrolling

## Short description
Large list of organizations in delegations are visible with scrolling mode.

## List of changes proposed in this pull request
- Added optional maxHeight parameter in component OrganizationsList in DelegationsElement file
- Extended tests for OrganizationsList

## How to test
Enter ada in DEV and check for delegations. You'll see the scrollbar in the organizations list now.
